front/src/components/Button/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import CreateCustomMaterialUlComponent from "utils/CreateCustomMaterialUlComponent";
+import ButtonMaterial from "@material-ui/core/Button";
+import * as Styles from "./styles";
+
+type Props = {
+  handleClick?: any;
+  styles?: any;
+  children: any;
+  customVariant?: string;
+  color?: string;
+  variant?: string;
+};
+
+const Button: React.FC<Props> = ({ handleClick, customVariant, ...rest }) => (
+  <CreateCustomMaterialUlComponent
+    handleClick={handleClick}
+    element={ButtonMaterial}
+    styles={
+      customVariant === "short"
```

Comment:
jezus maria, nigdy nie rob wiecej niz 1 ternary naraz, tego sie nie da odczytac

nizej komentarz jak to mozna ogarnac

Reply:
Done.